docs/payments/payment-gateway-api.md

```diff
@@ -27,7 +27,7 @@ Payment gateways should be created as additional plugins that hook into WooComme
 add_action( 'plugins_loaded', 'init_your_gateway_class' );
 ```
 
-It is also important that your gateway class extends the WooCommerce base gateway class, so you have access to important methods and the [settings API](https://woocommerce.com/document/settings-api/ "https://woocommerce.com/document/settings-api/"):
+It is also important that your gateway class extends the WooCommerce base gateway class, so you have access to important methods and the [settings API](https://developer.woocommerce.com/docs/settings-api/ "https://developer.woocommerce.com/docs/settings-api/"):
```

Comment:
This could be a good opportunity to remove the title that just repeats the URL from the link markdown -- or is there a special reason why we need that? 

```suggestion
It is also important that your gateway class extends the WooCommerce base gateway class, so you have access to important methods and the [settings API](https://developer.woocommerce.com/docs/settings-api/):
```

(Also see a few other links in this PR.)